setup.sh

```diff
@@ -239,6 +255,8 @@ if [ "$INSTALL_LOGLOADER" = "y" ]; then
 
 	make install
 
+	sudo cp build/logloader /usr/local/bin/logloader
```

Comment:
Why are you copying the contents of build? The `make install` above already takes care of installing

Reply:
Doesn't appear that make install is updating the binary.

Reply:
I pushed a logloader commit which performs a clean build on `make install`. Should fix the issue